src/bulkExport/stopExportJob.ts

```diff
@@ -31,7 +34,5 @@ export const stopExportJobHandler: Handler<BulkExportStateMachineGlobalParameter
         logger.error('Failed to stop job', JSON.stringify(stopJobRunResponse));
         throw new Error(`Failed to stop job ${glueJobRunId}`);
     }
-    return {
-        jobId: event.jobId,
-    };
+    return event;
```

Comment:
It doesn't look like we are modifying `event` at all do we need to return it?

Reply:
Yes, we need to return it as the input for the next step in the state machine for DDB status update. Before multi-tenancy, the next step only needs jobId, now it needs both tenantId and jobId. Passing tenantId as `tenantId: event.tenantId` would error out the state machine if event does not have tenantId attribute, so I'm passing the event altogether.